quantus/helpers/model/pytorch_model.py

```diff
@@ -219,3 +219,67 @@ def sample(
                             "or 'additive' (string) when you sample the model."
                         )
         return model_copy
+
+    def get_hidden_representations(
+        self,
+        x: np.ndarray,
+        layer_names: Optional[List[str]] = None,
+        layer_indices: Optional[List[int]] = None,
+    ) -> np.ndarray:
+
+        device = self.device if self.device is not None else "cpu"
+        all_layers = [*self.model.named_modules()]
+        num_layers = len(all_layers)
+
+        if layer_indices is None:
+            layer_indices = []
+
+        # Convert negative indices to positive
```

Comment:
Sorry, but for code cleanliness, all comments should be ended with a dot.

Reply:
done